addons/expanding-search-bar/expanding-search-bar.js

```diff
@@ -0,0 +1,61 @@
+export default async function ({ addon, global, console }) {
+	if (addon.tab.clientVersion == null) return; //if neither www or r2, exit
+	
+	var exsearch_searchBar; //The search bar element
+	
+	///Events
+	if (addon.tab.clientVersion == "scratch-www") { //We're on scratch-www
+		
+		//Elements
+		exsearch_searchBar = document.getElementById("frc-q-1088"); //The scratch-www search bar
+		var exsearch_links; //Header links
+
+		//Functions
+		function exsearch_getLinks() { //Gets all header links
+			let e = document.getElementsByClassName("link")[0]; //The first link
+			let list = []; //The list
+			
+			while (e.classList.contains("link")) { //Repeat until the element is NOT a link
+				list.push(e); //Add
+				e = e.nextSibling; //Next 
+			}
+			return list;
```

Comment:
You can just query `.link:not(.search ~ *)` and probably just switch to using css

Reply:
Also done. I don't think switching entirely to a style would be simple.